packages/ia-components/sandbox/theatres/with-youtube-spotify/utils/utils.test.js

```diff
@@ -59,4 +61,26 @@ describe('Music Player Util - General', () => {
       expect(isValidImageFile('image-title.jpeg')).toBe(true);
     });
   });
+  describe('Returns true when filename has the OLDER derived MP3 extension', () => {
```

Comment:
Might be a good idea to ensure we're only catching `/_vbr.mp3$/` by asserting something like 'track_vbr.mp3-track.mp3' returns false.

Reply:
thanks, added.